File: app/src/main/java/com/example/kocja/rabbiter_online/broadcastrecievers/ServiceStarterBroadCReciever.java
package com.example.kocja.rabbiter_online.broadcastrecievers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.os.Build;

public class ServiceStarterBroadCReciever extends BroadcastReceiver {
    @Override
    public void onReceive(Context context, Intent intent) {

    }
    public static void startService(Context c,Intent notifyService){
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O){
            c.startForegroundService(notifyService);
        }
        else{
            c.startService(notifyService);
        }
    }
}

File: app/src/main/java/com/example/kocja/rabbiter_online/broadcastrecievers/ServiceStarterBroadCReciever.kt
package com.example.kocja.rabbiter_online.broadcastrecievers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.os.Build

open class ServiceStarterBroadCReciever : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {

    }

    companion object {
        fun startService(c: Context, notifyService: Intent) {
            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
                c.startForegroundService(notifyService)
            } else {
                c.startService(notifyService)
            }
        }
    }
}
